modules/social_features/social_profile/social_profile.install

```diff
@@ -551,15 +551,44 @@ function social_profile_update_11002(): string {
   return $updateHelper->logger()->output();
 }
 
+/**
+ * Dummy update hook to prevent discrepancy.
+ */
+function social_profile_update_11003(array &$sandbox): void {
+  return;
```

Comment:
There is no need to add a "return" statement since this function is declaring "void" as return type ;) 

How about leaving an empty comment like this:

// Empty hook update.

I would suggest the same for the other hooks.